ادابتير ديزاين باترين/ادابتير ديزاين باترين/عدو مميز.cs

```diff
@@ -0,0 +1,11 @@
+﻿namespace ادابتير_ديزاين_باترين
+{
+    public class عدو_مميز
+    {
+        public string القاء_تعويذه()
+        {
+            return "using spell";
```

Comment:
The question is, why did you use English here while your code is all in Arabic 🤓